src/lib.rs

```diff
@@ -465,8 +465,6 @@ macro_rules! quick_error {
         queue [ #[$qmeta:meta] $( $tail:tt )*]
     ) => {
         quick_error!(SORT [$( $def )*]
-            enum [$( $(#[$emeta])* => $eitem $(( $($etyp),* ))* )*
-                     $(#[$bmeta])* => $bitem: $bmode $(( $($btyp),* ))*]
```

Comment:
I'm not sure I understand this deletion. Is this just the unused code that sneaked into the codebase?

Reply:
I am not familiar with the code, but this rule is very broken. Here is a similar fix: https://github.com/rust-lang-nursery/error-chain/pull/269/files#r305615630. Ideally, someone who understands the code should review this. I'm also fine just reverting the pull request if it can't be reviewed.

Reply:
Aha, this is left from @colin-kiegel's simplification commit: 94f0680
The code that matches that rule is:
```
pub enum Test {
    #[cfg(feature = "foo")]
    Variant1
    #[cfg(feature = "foo")]
    Variant2
}
```
Will try to fix it soon.

How did you find these issues?

Reply:
Nice find!

I've found those issues with a [crater run](https://github.com/rust-lang/rust/pull/62008#issuecomment-512183546) when adding the [`meta_variable_misuse` lint](https://github.com/rust-lang/rust/pull/62008). You can try it with `#![deny(meta_variable_misuse)]` (or `warn` instead of `deny`).

Reply:
I see. Thanks! I'm going to merge it as is and tackle the bug afterwards.

Reply:
Ok, thanks!